.github/workflows/runtime-builder-image.yml

```diff
@@ -0,0 +1,66 @@
+# NOTE: This name appears in GitHub's Checks API and in workflow's status badge.
+name: runtime-builder-image
+
+on:
+  push:
+    # XXX: ideally on main branches we would build the image only if there are changes in the
+    # 'docker/' directory (as we do in pull_requests). However, this doesn't work when pushing a new
+    # 'stable/*' branch - the build on a new branch does not trigger unless there are changes
+    # compared to main on the filtered path.
+    # If this is ever fixed, or per branch filters are possible, bring back the path filter to only
+    # build the image when there are changes within 'docker/' directory.
+    branches:
+      - main
+      - stable/*
+    paths:
+      - docker/runtime-builder/**
+  # Or when a pull request event occurs for a pull request against one of the matched branches and at least
+  # one modified file matches the configured paths.
+  #
+  # NOTE: We use this to be able to easily test Docker image changes.
+  pull_request:
+    branches:
+      - main
+      - stable/*
+    paths:
+      - docker/runtime-builder/**
+  # Or every day at 04:00 UTC (for the default/main branch).
+  schedule:
+    - cron: "0 4 * * *"
+
+jobs:
+
+  build-runtime-builder:
+    # NOTE: This name appears in GitHub's Checks API.
+    name: build-runtime-builder
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          # Check out pull request's HEAD commit instead of the merge commit.
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Determine tag name
+        id: determine-tag
+        uses: ./.github/actions/determine-tag
```

Comment:
Can you also update https://github.com/oasisprotocol/oasis-sdk/blob/main/.github/workflows/docker.yml to use this internal action for determining tag?

---

Also maybe rename the existing `docker` workflow to `benchmarking-image` (or something similar) now that we have multiple "docker" workflows